common.mk

```diff
@@ -114,6 +114,12 @@ test-openshift: script_env += TEST_OPENSHIFT_MODE=true
 test-openshift: tag
 	VERSIONS="$(VERSIONS)" BASE_IMAGE_NAME="$(BASE_IMAGE_NAME)" $(script_env) $(test)
 
+.PHONY: test-openshift-pytest
+test-pytest: script_env += TEST_OPENSHIFT_PYTEST=true
```

Comment:
The makefile target is still called test-pytest, not test-openshift-pytest.
Also is there any reason for duplicating it?

Reply:
No. Stupid typo from me. Fixed by https://github.com/sclorg/container-common-scripts/pull/373/commits/d8cc7304106a0abe6bfdd65611cd9f601e0e90a4